pkg/xds/secrets/secrets_test.go

```diff
@@ -99,14 +101,26 @@ var _ = Describe("Secrets", func() {
 		}
 	}
 
+	BeforeAll(func() {
+		core_mesh.AllowedMTLSBackends = 2
```

Comment:
Now that we actually create a mesh it goes through validation and we have a validation to limit backends to one. I just bumped the limit so that the test works, maybe we could restructure it to work with 1 backend? Should we?

Reply:
Bumping the limit is fine IMO